src/components/Icon/README.md

```diff
@@ -22,40 +21,80 @@ Icons are used to visually communicate core parts of the product and available a
 
 ---
 
-## Best practices
+## Examples
+
+### Default icon
```

Comment:
Can you add `<!-- example-for: web -->` after both example titles?

Reply:
@kaelig I'm not quite sure where you mean. Do you mean for the code snippets in the Do and Don't usageblocks? I didn't alter the working examples.

Reply:
Adding the comment like so will ensure the examples don't appear under iOS and Android tabs in the style guide:

```suggestion
### Default icon

<!-- example-for: web -->
```

Reply:
Never mind - since all (both) examples are for web, this is not required!